Smoothing code.cpp

```diff
@@ -31,6 +31,31 @@ struct MovingAverage {
 std::uint16_t get_reading() {
     // this always returns 7 just because.... put your "read the pin" code here
     return 7;
+  int sensorPin = A0;
+int ledPin = LED_BUILTIN;
+int sensorValue = 0;
```

Comment:
all these pin variables need to move to the "global scope" as it's called.
so, where `THRESHOLD` and `MOV_AVG_SIZE` are -- this means everything can access it and makes it easy in this case (though in non-embedded systems it's bad practice, but your're fine here)